models/xai_models.py

```diff
@@ -1,73 +1,51 @@
 """
-$ pip install openai # we use openai api sdk for simplicity
+$ pip install openai # we use openai sdk for simplicty
```

Comment:
maybe reword to "Xai uses openai api for simplicity" - this is a bit ambiguous, I firstly thought we do it for simplicity and for some reason it works